HISTORY.md

```diff
@@ -1,7 +1,12 @@
+# 1.5.3
```

Comment:
Thanks for doing this, but in the future, don't worry about updating HISTORY.md -- I do it before every release :-)

You have 2 pending PRs, I'd prefer to roll them both out in one-shot if possible.